setup.py

```diff
@@ -74,7 +74,7 @@
     include_package_data=True,
     install_requires=[
         "hexbytes>=0.3.0,<1",
-        "pydantic>=1.10.7,<2",
+        "pydantic>=1.10.7",  # pydantic >=2 is supported
```

Comment:
Maybe cap on `<3`

Reply:
im not fan of upper caps, they always cause problems

Reply:
When they upgrade to v3, they will drop the v1 backports